src/main/default/lwc/createCaseButton/createCaseButton.js

```diff
@@ -87,6 +89,7 @@ export default class CreateCaseButton extends LightningElement {
   }
 
   async handleCreateCase() {
+    this.loading = true;
     const newCaseId = await newCase(this.caseData);
```

Comment:
If `newCase(this.caseData)` throws, the loading animation will stick. I would suggest to wrap the function body into a `try/finally` to make sure the error is logged properly and the animation does not stick.

Reply:
Thanks! nice point👌